test/functional/db.test.js

```diff
@@ -205,7 +205,7 @@ describe('Db', function() {
   /**
    * An example showing how to force a reindex of a collection.
    */
-  it('shouldCorrectlyForceReindexOnCollection', {
+  it.skip('shouldCorrectlyForceReindexOnCollection', {
```

Comment:
I think its safe to remove this if we're removing `reIndex` outright. The description shows us that the test is "An example showing how to force a reindex of a collection"

Reply:
ditto for the below tests